src/Libraries/Microsoft.Extensions.TimeProvider.Testing/FakeTimeProvider.cs

```diff
@@ -14,13 +15,16 @@ namespace Microsoft.Extensions.Time.Testing;
 /// </summary>
 public class FakeTimeProvider : TimeProvider
 {
-    internal static readonly DateTimeOffset Epoch = new(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
-
     internal readonly List<FakeTimeProviderTimer.Waiter> Waiters = new();
 
-    private DateTimeOffset _now = Epoch;
+    private DateTimeOffset _now;
     private TimeZoneInfo _localTimeZone;
 
+    /// <summary>
+    /// Gets the time at which the instance of the <see cref="FakeTimeProvider"/> was initialized.
+    /// </summary>
+    public DateTimeOffset StartTime { get; } = new DateTimeOffset(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
+
```

Comment:
I think the comment and the property name are misleading. This is not related to the time that this object was created, it is in fact a completely arbitrary value. I would rename the property back to Epoch and change the parameter name in the constructor to "epoch" to match.

Reply:
I'll leave it up to you, @sebastienros, and @RussKie to decide where to go with this. Here is the initial input I received for reference: https://github.com/dotnet/extensions/pull/4022#discussion_r1211974488

Reply:
I agree about the code docs being misleading. I've updated it. Let me know if you think this is better.

Reply:
Ok to rename this Epoch like @geeknoid is suggesting and keep it public/get. I can see now why this could be important to be able to set it.

Reply:
Should the argument passed to the ctor also be named `epoch` to align the two?